### PR TITLE
[FIX] Improved error handling

### PR DESCRIPTION
**Pull request by Valentin-Kaiser**
I recommend using `log.Fatal()` instead of `panic()` or `log.Panic()` for expected error cases that should terminate the process. Because exceptions thrown by `panic()` are always caught by the PanicHandler in the next step, most errors are not exception errors and should not be handled that way. 

For more information:

https://pkg.go.dev/log#Fatal
https://pkg.go.dev/log#Panic

### CODE REVIEW

File: pkg/importer/importer.go [hunk 1]
```diff
@@ -48,43 +56,36 @@ func (manager *Manager) Import() {
 
 	err := imports.ImportAgents(manager.Config.OMSDBPath)
 	if err != nil {
-		log.Print("[importer] importing agents failed")
-		panic(err.Error())
+		log.Printf("[importer] importing agents failed - error: %s", err.Error())
```
Comment (Plaenkler) [inline pkg/importer/importer.go:59]: Why `Fatal` everywhere and here only a `Print`?

File: pkg/importer/importer.go [hunk 2]
```diff
@@ -48,43 +56,36 @@ func (manager *Manager) Import() {
...
 	}
 
 	err = imports.ImportArticles(manager.Config.OMSDBPath)
 	if err != nil {
-		log.Print("[importer] importing articles failed")
-		panic(err.Error())
+		log.Fatalf("[importer] importing articles failed - error: %s", err.Error())
```
Comment (Plaenkler) [inline pkg/importer/importer.go:64]: The software should not be terminated if the import fails. There may already be data for the presentation.

**Review by Plaenkler — CHANGES_REQUESTED** [review]
I can't quite understand the choices where the process is terminated.

File: pkg/importer/importer.go (hunk 1 above)
Comment (Valentin-Kaiser) [reply pkg/importer/importer.go:59]: In my opinion, a failed agent import should not stop the application, but inform the user via a log. Furthermore, the user should be able to restart the import after it has failed. A failed import can have many different reasons, such as current network problems that are fixed at a later time. The application works even if the data is not up to date.

Comment (Valentin-Kaiser) [reply pkg/importer/importer.go:59]: You even answered it yourself in the next thread! xD

File: pkg/importer/importer.go (hunk 2 above)
Comment (Valentin-Kaiser) [reply pkg/importer/importer.go:64]: You're right about that. See previous thread.

**Comment by Valentin-Kaiser** [comment]
> I can't quite understand the choices where the process is terminated.

I'm sure you can, see the threads. In my opinion, there is no need for changes.

**Review by Plaenkler — APPROVED** [review]

**Pull request merged**